### ISSUE TITLE
Rule Proposal: warn if ElementFinder methods are not called in expect()

### ISSUE THREAD

**Issue by alecxe** [alecxe/eslint-plugin-protractor#84]
This should be a warning:

```js
expect($(".myclass").isDisplayed).toBe(true);
```

Not a warning:

```js
expect($(".myclass").isDisplayed()).toBe(true);
```
